controllers/comment.js

```diff
@@ -0,0 +1,155 @@
+import { Comment, User } from '../models';
+
+/**
+ * @class CommentController
+ *  @override
+ * @export
+ *
+ */
+export default class CommentController {
+  /**
+   * @description - Creates an article comment
+   * @static
+   *
+   * @param {object} req - HTTP Request
+   * @param {object} res - HTTP Response
+   *
+   * @memberof CommentController
+   *
+   * @returns {object} Article comment
+   */
+  static async createComment(req, res) {
+    const { id } = req.user;
+    const { slug: articleSlug } = req.params;
```

Comment:
@olajide1234 I'm not sure the param used by others, please confirm and ensure it's uniform.

Reply:
I have confirmed, the team uses `req.params.slug` to pass the slug of articles. I have only reassigned the name to `articleSlug` in my controller.